src/app/options.cpp

```diff
@@ -247,6 +315,13 @@ namespace
     constexpr const StringOption CONFIGURATION_OPTION = {"configuration"};
     constexpr const BoolOption PORTABLE_OPTION = {"portable"};
     constexpr const BoolOption RELATIVE_FASTRESUME = {"relative-fastresume"};
+    constexpr const StringOption SAVE_PATH_OPTION = {"save-path"};
+    constexpr const TriStateBoolOption PAUSED_OPTION = {"add-paused"};
+    constexpr const BoolOption SKIP_HASH_CHECK_OPTION = {"skip-hash-check"};
+    constexpr const StringOption CATEGORY_OPTION = {"category"};
+    constexpr const BoolOption SEQUENTIAL_OPTION = {"sequential"};
+    constexpr const BoolOption FIRST_AND_LAST_OPTION = {"first-and-last"};
```

Comment:
@sledgehammer999 What about cases where qBittorrent's application-wide settings are configured to a) add torrents paused by default or b) skip the dialog by default? There should be a way to override the application settings from the command line and force adding a torrent in a started state or force showing the "Add new torrent" dialog, hence the ability to specify `--add-paused=false` or `--skip-dialog=false`.

We eventually settled on having a tri-state bool command line option since it seemed like the best solution, especially since the options for whether a new torrent should be started or paused are mutually exclusive.